app/services/samples/update_service.rb

```diff
@@ -13,7 +13,20 @@ def initialize(sample, user = nil, params = {})
     def execute
       authorize! sample.project, to: :update_sample?
 
-      sample.update(params)
+      sample_updated = sample.update(params)
+
+      if sample_updated
+        sample.project.namespace.create_activity key: 'namespaces_project_namespace.samples.update',
+                                                 owner: current_user,
+                                                 parameters:
+                                                  {
+                                                    sample_id: sample.id,
+                                                    sample_name: sample.puid,
```

Comment:
**nitpick (blocking):** Please rename this to `sample_puid`

Reply:
Updated in 032cb1c08ee993dee5d18b74ef198b03bafe07b3